CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `AuthorityFactory`: Allows anyone to deploy `Authority` contracts. Supports deterministic deployment.
+
```

Comment:
Should we add this comments in the `Changelog` manually again?

Reply:
Yes, we've opted to change the `CHANGELOG` manually per PR.

@gligneul has realized that we can't rely on conventional commits only. There is still manual work of filtering and tailoring commit messages. So, it's better to spread this work per PR rather than concentrate it all in release time.